test/example.js

```diff
@@ -1,9 +1,9 @@
 'use strict'
 
 var test = require('tape')
-var is = require('.')
+var is = require('..')
 
-test('unist-util-is', function (t) {
+test('unist-util-is example', function (t) {
```

Comment:
example may not be the best name for the tests that give 100% coverage 😅

Reply:
:man_shrugging: there are two types of tests in the test suite now, example based tests and property based tests.
Is there another way you would prefer these to be named?

Reply:
I like `core`, `main`, or so, because they’re the unit tests that provide the coverage.

`example` to me indicates that it checks one test, probably from the readme Use section.

Reply:
pluralized https://github.com/ChristianMurphy/unist-util-is/blob/7170eb9d35e8b08dbaae297eae9c320665b7fa05/test/example.js#L6

>  like core, main, or so, because they’re the unit tests that provide the coverage.

:raised_eyebrow: the properties tests also provide coverage, do you want the excluded from coverage?

Reply:
Sure they do, but when something changes in the library, a feature or a bug fix, that test is likely going to go in the main test file. It *may* be added to properties too. But I don’t expect folks working on their first PR to do so, nor to guess that `examples` is where the main tests should go

Reply:
Okay, updated in e10c23cddc0a4b6899c9137e862ad87d3a7a1ed9